pages/api/utils/policy-utils.js

```diff
@@ -0,0 +1,40 @@
+/**
+ * Copyright 2021 The Rode Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+export const mapToClientModel = (policyResponse) => {
+  return {
+    id: policyResponse.id,
+    name: policyResponse.name,
+    description: policyResponse.description,
+    regoContent: policyResponse.policy.regoContent,
+  };
+};
+
+export const mapToApiModel = (requestBody) => {
+  let formattedRequest = requestBody;
+
+  if (typeof requestBody === "string") {
```

Comment:
I was wondering why Next.js wasn't parsing `req.body` automatically, as the [docs](https://nextjs.org/docs/api-routes/api-middlewares) suggest that it should happen.

I think the issue is that the client side fetch isn't specifying the content type of the request. I updated the fetch call in `<PolicyForm />` to add the content type:

```javascript
    const response = await fetch(endpoint, {
      method,
      headers: {
        'Content-Type': 'application/json',
      },
      body: JSON.stringify(formData),
    });
```

Then I saw that the type of `req.body` was an object in the API handler